cli/azd/pkg/output/ux/displayed_resource.go

```diff
@@ -35,11 +38,17 @@ func (cr *DisplayedResource) ToString(currentIndentation string) string {
 		prefix = donePrefix
 	}
 
-	return fmt.Sprintf("%s%s %s: %s", currentIndentation, prefix, cr.Type, cr.Name)
+	var builder strings.Builder
+	builder.WriteString(fmt.Sprintf("%s%s %s: %s", currentIndentation, prefix, cr.Type, cr.Name))
+
+	if cr.Duration.Seconds() > 5.0 {
```

Comment:
I had to pick some number here and so I pickled 5 seconds.  I could imagine that there are better choices (for example, in the provided screenshot we show 7 seconds, which still feels "fast" to me).  Maybe 30 seconds?

I wanted to set some line because every resource has some amount of time, but I feel like tools look silly when they show small durations like this for what is part of an overall a long operation.

Reply:
I think the intention here may be to highlight long-running operations. If so, 1 minute may be a good threshold.

I do think it's better we have a higher threshold for now. We want to avoid reporting on more than half of the operations. If we do report on many, the user may think it's accurate inclusive time distribution and gets confused when the individual operation numbers don't agree with the sum numbers.